storage/namespace.go

```diff
@@ -373,6 +374,9 @@ func (n *dbNamespace) Repair(repairer databaseShardRepairer) error {
 			numChecksumDiffSeries += metadataRes.ChecksumDifferences.NumSeries()
 			numChecksumDiffBlocks += metadataRes.ChecksumDifferences.NumBlocks()
 		}
+		if throttle > 0 {
+			time.Sleep(throttle)
```

Comment:
Maybe we should have a throttle value that is fixed and we divide it by how many shards we own to get the value for this sleep here?  That way we'll get a known amount of sleeping no matter how many shards a node might own.